utils/schema/core-datasource-ids.json

```diff
@@ -0,0 +1,140 @@
+[
```

Comment:
Out of curiosity, are all these core data sources visibile to all customers?

Unsure if there was already discussion about only allowing core data sources that have their `visibility` flags set to `deployed` / 100% traffic. 

Not a blocking comment, and since we call it out in the docs to ensure that core data sources are fully deployed and visible, I see this more on the contributor to do their due-diligence.

Reply:
Memory is a bit hazy, but I think we ran some scripts to get all the IDs and we _might_ have factored in the existence of a feature flag.

That said, short of clicking into each feature flag, I'm not sure how we would be able to verify the traffic %. Totally open to suggestions on how to improve tho!

Reply:
Top of my head i cant see think of a way without creating some type of script which _could_ be over-engineer to figure out this "simple" edge case.

I think it could be something to look into at another point, but def not worth the scope of this ticket. Maybe once we get some sort of validaiton on the elixir side we could add it, but not sure if we could necessarily catch it at PR time.

Reply:
Flags weren't taken into account so this is just all core data sources.  I imagine its fine to show the IDs, flags being more for hiding the experience or visibility.  Feature flags also generally contain the data source id (in most cases) and are visible in the network.  Since this is schema related and not building a list for the client, don't we need all core data source IDs to be present?